userspace/h1b_tests/Build.mk

```diff
@@ -57,18 +57,27 @@ userspace/h1b_tests/run: build/cargo-host/release/runner \
 build/userspace/h1b_tests/h1b_tests:
 	rm -f build/userspace/cargo/thumbv7m-none-eabi/release/h1b_tests-*
 	cd userspace/h1b_tests && TOCK_KERNEL_VERSION=h1b_tests \
-		cargo test --no-run --release -Z offline
+		cargo test --no-run --offline --release
 	mkdir -p build/userspace/h1b_tests/
 	find build/userspace/cargo/thumbv7m-none-eabi/release/ -maxdepth 1 -regex \
 		'build/userspace/cargo/thumbv7m-none-eabi/release/h1b_tests-[^.]+' \
 		-exec cp '{}' build/userspace/h1b_tests/h1b_tests ';'
 
+# Due to b/139156455, we want to detect the case where an application's size is
+# rounded up to 64 KiB.
 build/userspace/h1b_tests/h1b_tests.tbf: \
 		build/cargo-host/release/elf2tab build/userspace/h1b_tests/h1b_tests
 	build/cargo-host/release/elf2tab -n "h1b_tests" \
 		-o build/userspace/h1b_tests/h1b_tests_tab \
 		build/userspace/h1b_tests/h1b_tests --stack=2048 --app-heap=1024 \
 		--kernel-heap=1024 --protected-region-size=64
+	if [ "$$(wc -c <build/userspace/h1b_tests/h1b_tests.tbf)" -ge 65536 ]; \
```

Comment:
Rather than hard-coding this 65536 here, I'd suggest pulling it from chip_layout.